### PR TITLE
Make tests pass at any timezone

### LINKED ISSUE #43: Unit tests fail when computer is on a different timezone than Western Europe
Many unit tests rely that the "Create At", "Updated At" and zettelkasten IDs will be rendered according  to the user's timezone, which may differ while running unit tests.

Tests could use a library like timezone-mock to ensure they always run on a given timezone and therefore always produce the same expected output.

### PR DESCRIPTION
**Pull request by rodbv**
Fixes #43 

Changed fixture MD files so that they expect times on Europe/London timezone (offset +01:00), and mocked this timezone on yarle.spec.ts.

This ensures that these tests will pass no matter what is the timezone of the computer running the tests.

### CODE REVIEW

**Comment by rodbv** [comment]
Also fixed issues with the 2 tests that had it.skip. All tests running and passing now! 👍

File: package.json [hunk 1]
```diff
@@ -64,6 +65,7 @@
     "@types/sinon": "9.0.1",
     "mocha": "5.2.0",
     "nyc": "13.0.1",
-    "sinon": "9.0.2"
+    "sinon": "9.0.2",
+    "timezone-mock": "^1.1.1"
```
Comment (akosbalasko) [inline package.json:69]: Please remove hat from the dependency, just to prohibit that a breaking change in a future-version of timezone breaks the conversion. Thank you!

```suggestion
    "timezone-mock": "1.1.1"
```

File: test/yarle.spec.ts [hunk 2]
```diff
@@ -8,372 +9,621 @@ import * as yarle from './../src/yarle';
...
-                     fs.readFileSync(`${__dirname}/data/test-obsidianLink.md`, 'utf8'));
-        assert.equal(fs.existsSync(`${__dirname}/../out/simpleNotes/test-twoNotes/test -note with text only.md`), true);
-
-        assert.equal(fs.readFileSync(`${__dirname}/../out/simpleNotes/test-twoNotes/test -note with text only.md`, 'utf8'),
-                    fs.readFileSync(`${__dirname}/data/test-twoNotes-text.md`, 'utf8'));
-
-    });
-
+  before(() => {
+    mockTimezone.register('Europe/London');
```
Comment (akosbalasko) [inline test/yarle.spec.ts:13]: Thank you for this, I didn't know this package.

**Review by akosbalasko — CHANGES_REQUESTED** [review]
Hi! 
First of all, thank you for your fork + fixes, I really appreciate them! 
I've got only one short modification request, as you see, then I'm glad to merge your fix!
Cheers,
akos

**Review by akosbalasko — APPROVED** [review]

**Pull request merged**